data/chromeos.ini

```diff
@@ -0,0 +1,59 @@
+[DEFAULT]
+branch: chromeos.kernelci.org
+namespace: kernelci
+users:
```

Comment:
We might want to trim this down to the list of users who are actually going to be working on the chromeos.kernelci.org instance.

Reply:
I wanted to clarify that during tonight's Open Hours. I'll clear the users list.

Reply:
I reduced the user list